examples/sft_sentiments.py

```diff
@@ -1,26 +1,20 @@
 import os
-import pathlib
 from typing import Dict, List
 
-import yaml
 from datasets import load_dataset
 from transformers import pipeline
 
 import trlx
-from trlx.data.configs import TRLConfig
-
-config_path = pathlib.Path(__file__).parent.joinpath("../configs/sft_config.yml")
-with config_path.open() as f:
-    default_config = yaml.safe_load(f)
+from trlx.data.default_configs import default_sft_config
 
 
 def get_positive_score(scores):
     "Extract value associated with a positive sentiment from pipeline's output"
     return dict(map(lambda x: tuple(x.values()), scores))["POSITIVE"]
 
 
-def main(hparams={}):
-    config = TRLConfig.update(default_config, hparams)
```

Comment:
Both hyperparamer optimization (wandb's and ray tune's) relied on this config propagation, either through passing parameters updates on cli or by importing main directly and passing it within an interpreter. Not sure is there a better way of doing it, althought I feel that something emulating hydra's `+` operator is still essential

Reply:
Ah I see, I'll add them back. Yeah, mergability is easy to add back.